DEV-3/DEV-3/ArgumentsChecker.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace DEV_3
+{
+    /// <summary>
+    /// This Class checks arguments to be corresponding.
+    /// </summary>
+    class ArgumentsChecker
+    {
+        /// <summary>
+        /// This Method checks string arguments to be corresponding.
+        /// </summary>
+        /// <param name="args">Initial arguments.</param>
+        public void CheckStringArgs(string[] args)
+        {
+            if (args.Length != 2)
+            {
+                Console.WriteLine("Wrong number of arguments.");
+                Environment.Exit(0);
+            }
+            int result1, result2 = 0;
+            if (!Int32.TryParse(args[0], out result1) || !Int32.TryParse(args[1], out result2))
+            {
+                Console.WriteLine("Wrong type of arguments.");
+                Environment.Exit(0);
```

Comment:
never use exit as it's extremely unpredictable way to terminate application for method caller